test/unit-test/rtp_packet_queue/rtp_packet_queue_utest.c

```diff
@@ -0,0 +1,434 @@
+                                              &( rtpPacketInfo ),
+                                              &( deletedRtpPacketInfo ) );
+
+        TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_OK, result );
+    }
+
+    result = RtpPacketQueue_ForceEnqueue( &( rtpPacketQueue ),
+                                          &( rtpPacketInfo ),
+                                          &( deletedRtpPacketInfo ) );
+
+    TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_PACKET_DELETED, result );
+    TEST_ASSERT_EQUAL( 0, deletedRtpPacketInfo.seqNum ); /* First packet deleted. */
+
+    result = RtpPacketQueue_ForceEnqueue( &( rtpPacketQueue ),
+                                          &( rtpPacketInfo ),
+                                          &( deletedRtpPacketInfo ) );
+
+    TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_PACKET_DELETED, result );
+    TEST_ASSERT_EQUAL( 1, deletedRtpPacketInfo.seqNum ); /* Second packet deleted. */
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Validate dequeue when queue is empty.
+ */
+void test_RtpPacketQueue_Dequeue_EmptyQueue( void )
+{
+    RtpPacketQueueResult_t result;
+    RtpPacketInfo_t rtpPacketInfo;
+
+    result = RtpPacketQueue_Init( &( rtpPacketQueue ),
+                                  &( rtpPacketInfoArray[ 0 ] ),
+                                  MAX_IN_FLIGHT_PKTS );
+
+    TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_OK, result );
+
+    result = RtpPacketQueue_Dequeue( &( rtpPacketQueue ),
+                                     &( rtpPacketInfo ) );
+
+    TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_EMPTY, result );
+}
+
+/*-----------------------------------------------------------*/
+
+/**
+ * @brief Validate dequeue functionality.
+ */
+void test_RtpPacketQueue_Dequeue( void )
+{
+    RtpPacketQueueResult_t result;
+    RtpPacketInfo_t deletedRtpPacketInfo, rtpPacketInfo;
+    uint8_t expectedSerializedPacket[] = { 0x80, 0x66, 0xAB, 0x12,
+                                           0x12, 0x34, 0x43, 0x21,
+                                           0xAB, 0xCD, 0xDB, 0xCA };
+
+    result = RtpPacketQueue_Init( &( rtpPacketQueue ),
+                                  &( rtpPacketInfoArray[ 0 ] ),
+                                  MAX_IN_FLIGHT_PKTS );
+
+    TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_OK, result );
+
+    rtpPacketInfo.seqNum = 10;
+    rtpPacketInfo.pSerializedRtpPacket = &( expectedSerializedPacket[ 0 ] );
+    rtpPacketInfo.serializedPacketLength = sizeof( expectedSerializedPacket );
+
+    result = RtpPacketQueue_ForceEnqueue( &( rtpPacketQueue ),
+                                          &( rtpPacketInfo ),
+                                          &( deletedRtpPacketInfo ) );
+
+    TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_OK, result );
+
+    result = RtpPacketQueue_Dequeue( &( rtpPacketQueue ),
+                                     &( rtpPacketInfo ) );
+
+    TEST_ASSERT_EQUAL( 10, rtpPacketInfo.seqNum );
```

Comment:
```suggestion
    TEST_ASSERT_EQUAL( RTP_PACKET_QUEUE_RESULT_OK, result );
    TEST_ASSERT_EQUAL( 10, rtpPacketInfo.seqNum );
```
I suggest to check the result here.